src/sidebar/components/test/SelectionTabs-test.js

```diff
@@ -49,54 +58,130 @@ describe('SelectionTabs', function () {
-    it('should display the tabs and counts of annotations and notes', function () {
-      const wrapper = createComponent();
-      const tabs = wrapper.find('button');
-      assert.isTrue(tabs.at(0).contains('Annotations'));
-      assert.equal(tabs.at(0).find('.SelectionTabs__count').text(), 123);
-      assert.isTrue(tabs.at(1).contains('Page Notes'));
-      assert.equal(tabs.at(1).find('.SelectionTabs__count').text(), 456);
-    });
+  it('should display the tabs and counts of annotations and notes', () => {
+    const wrapper = createComponent();
+    const tabs = wrapper.find('button');
+
+    assert.include(tabs.at(0).text(), 'Annotations');
+    assert.equal(tabs.at(0).find('.SelectionTabs__count').text(), 123);
 
-    it('should display annotations tab as selected', function () {
+    assert.include(tabs.at(1).text(), 'Page Notes');
+    assert.equal(tabs.at(1).find('.SelectionTabs__count').text(), 456);
+  });
+
+  describe('Annotations tab', () => {
+    it('should display annotations tab as selected when it is active', () => {
       const wrapper = createComponent();
+
       const tabs = wrapper.find('button');
+
       assert.isTrue(tabs.at(0).hasClass('is-selected'));
       assert.equal(tabs.at(0).prop('aria-selected'), 'true');
       assert.equal(tabs.at(1).prop('aria-selected'), 'false');
     });
 
-    it('should display notes tab as selected', function () {
+    it('should not display the add-page-note button when the annotations tab is active', () => {
+      fakeSettings.enableExperimentalNewNoteButton = true;
+      const wrapper = createComponent();
+      assert.equal(wrapper.find('LabeledButton').length, 0);
+    });
+  });
+
+  describe('Notes tab', () => {
+    it('should display notes tab as selected when it is active', () => {
       fakeStore.selectedTab.returns('note');
-      const wrapper = createComponent({});
+      const wrapper = createComponent();
+
       const tabs = wrapper.find('button');
+
       assert.isTrue(tabs.at(1).hasClass('is-selected'));
       assert.equal(tabs.at(1).prop('aria-selected'), 'true');
       assert.equal(tabs.at(0).prop('aria-selected'), 'false');
     });
 
-    it('should display orphans tab as selected if there is 1 or more orphans', function () {
+    describe('Add Page Note button', () => {
+      it('should not display the add-page-note button if the associated setting is not enabled', () => {
+        fakeSettings.enableExperimentalNewNoteButton = false;
+        fakeStore.selectedTab.returns('note');
+
+        const wrapper = createComponent();
+
+        assert.equal(wrapper.find('LabeledButton').length, 0);
+      });
+
+      it('should display the add-page-note button when the associated setting is enabled', () => {
+        fakeSettings.enableExperimentalNewNoteButton = true;
+        fakeStore.selectedTab.returns('note');
+
+        const wrapper = createComponent();
+
+        assert.equal(wrapper.find('LabeledButton').length, 1);
```

Comment:
A purely personal alternative:

```suggestion
        assert.isFalse(wrapper.find('LabeledButton').exists());
```